chains/ethereum/primitives/src/block.rs

```diff
@@ -0,0 +1,85 @@
+use crate::bytes::Bytes;
+use alloc::vec::Vec;
+use ethereum_types::{Address, Bloom, H256, H64, U256, U64};
+
+/// The block type returned from RPC calls.
+///
+/// This is generic over a `TX` type which will be either the hash or the full transaction,
+/// i.e. `Block<TxHash>` or `Block<Transaction>`.
+#[derive(Debug, Default, Clone, PartialEq, Eq)]
+#[cfg_attr(feature = "with-codec", derive(parity_scale_codec::Encode, parity_scale_codec::Decode))]
+#[cfg_attr(
+    feature = "with-serde",
+    derive(serde::Serialize, serde::Deserialize),
+    serde(rename_all = "camelCase")
+)]
+pub struct Block<TX> {
+    /// Hash of the block
+    pub hash: H256,
```

Comment:
I had to re-implement some [ethers-rs](https://github.com/gakonst/ethers-rs/blob/master/ethers-core/src/types/block.rs#L15-L113) primitives because they didn't implement scale-codec and scale-info traits.

I will use the [scale-info](https://github.com/paritytech/scale-info) to generate the metadata for allowing timegraph and other components to parse the encoded task.